docs/learning-powershell/using-vscode.md

```diff
@@ -16,46 +16,46 @@ Editing with Visual Studio Code
 
 * **Linux**: follow the installation instructions on the [Running VS Code on Linux](https://code.visualstudio.com/docs/setup/linux) page
 
-* **macOS**: follow the installation instructions on the [Running VS Code on macOS](https://code.visualstudio.com/docs/setup/osx) page
+* **macOS**: follow the installation instructions on the [Running VS Code on macOS](https://code.visualstudio.com/docs/setup/mac) page
 
-  **NOTE:** On OS X you must install OpenSSL for the PowerShell extension to work correctly.  The easiest way to
-            accomplish this is to install [Homebrew](http://brew.sh/) and then run `brew install openssl`.  The PowerShell extension
-            will now be able to load successfully.
+  **NOTE:** On OS X you must install OpenSSL for the PowerShell extension to work correctly.
+            The easiest way to accomplish this is to install [Homebrew](http://brew.sh/) and then run `brew install openssl`.
+            The PowerShell extension will now be able to load successfully.
 
 * **Windows**: follow the installation instructions on the [Running VS Code on Windows](https://code.visualstudio.com/docs/setup/windows) page
 
 
 **2. Installing PowerShell Extension**
 
--	Launch the Visual Studio Code app by:    
-  *	**Windows**:      typing **code** in your PowerShell session
-  *	**Linux**:        typing **code .** in your terminal
-  *	**macOS**:         typing **code** in your terminal
+-	Launch the Visual Studio Code app by:
+    *	**Windows**:      typing **code** in your PowerShell session
+    *	**Linux**:        typing **code .** in your terminal
```

Comment:
Perhaps Code required a path on Linux previously, but that doesn't seem the be the case anymore.  just `code` is fine for Linux